genesyscloud/tfexporter/genesyscloud_resource_exporter.go

```diff
@@ -246,24 +232,24 @@ func (g *GenesysCloudResourceExporter) retrieveGenesysCloudObjectInstances() dia
 	return nil
 }
 
-// buildJsonConfigMap Builds a map of all the JSON data returned for each resource
-func (g *GenesysCloudResourceExporter) buildJsonConfigMap() diag.Diagnostics {
-	log.Printf("Build JSON Map and HCL Blocks")
-	g.resourceTypeJSONMaps = make(map[string]map[string]gcloud.JsonMap)
```

Comment:
This was a small nit.  We are not really dealing with a JSON map.  We are dealing with a map of different resource types.  The map is used to generate JSON or HCL so I renamed the attribute